### DISCUSSION TITLE
%E4%BD%BF%E7%94%A8%E6%8C%87%E5%8D%97/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#44] [Announcements]
# %E4%BD%BF%E7%94%A8%E6%8C%87%E5%8D%97/

https://csmsapp.github.io/%E4%BD%BF%E7%94%A8%E6%8C%87%E5%8D%97/

**Comment by ghost** [comment]
求其上者得其中

**Comment by dercaft** [comment]
请问因素里面的Co-op是啥意思呀

**Reply by cindyuc** [reply]
Cooperative Education. 简单来说，对于F1签证的学生，可以选择在秋季/春季学期用CPT去实习。一般大家都是summer intern。 co-op相当于多了一次实习机会，能让你的简历更丰富，提高上岸率。可以参考：https://www.khoury.northeastern.edu/information-for-overview/current-masters-and-certificate-students/masters-co-op-internship-overview/masters-co-op-internship-policies/

**Reply by xichenpan** [reply]
美国有Co-op的学校我了解的是NEU GaTech TAMU
加拿大很多学校有，比如Waterloo MMath/MDSAI; UofT MScAc; UBC ECE Meng; SFU CS MSc等等

**Reply by json13245** [reply]
那请问🇨🇦像UofT/Waterloo ece MEng 不带co-op的是不是同时也不能 summer intern呢

**Reply by xichenpan** [reply]
这个是scheduled break，据我了解uoft和loo的ece meng没有scheduled break。
btw，昨天听人说loo的ece meng有学生收到coop的邀请了

**Reply by json13245** [reply]
感谢~
